app/assets/javascripts/admin/bulk_order_management.js.coffee

```diff
@@ -162,6 +163,12 @@ angular.module("ofn.admin").controller "AdminOrderMgmtCtrl", [
       $scope.supplierFilter = $scope.suppliers[0].id
       $scope.orderCycleFilter = $scope.orderCycles[0].id
       $scope.quickSearch = ""
+
+    $scope.weightAdjustedPrice = (lineItem, oldValue) ->
+      if lineItem.unit_value <= 0
+        lineItem.unit_value = lineItem.units_variant.unit_value
+      lineItem.price = lineItem.price * lineItem.unit_value / oldValue
```

Comment:
Oh yeah, good point. I had this logic and then I must have 'cleaned it up'...
Adding back in now. I think this is a better solution than populating on a migration... do you think?